trino/trino.go

```diff
@@ -290,11 +296,17 @@ func newConn(dsn string) (*Conn, error) {
 		if httpClient == nil {
 			return nil, fmt.Errorf("trino: custom client not registered: %q", clientKey)
 		}
-	} else if certPath := query.Get(SSLCertPathConfig); certPath != "" && serverURL.Scheme == "https" {
-		cert, err := ioutil.ReadFile(certPath)
-		if err != nil {
-			return nil, fmt.Errorf("trino: Error loading SSL Cert File: %v", err)
+	} else if serverURL.Scheme == "https" {
+
+		cert := []byte(query.Get(SSLCertConfig))
+
+		if certPath := query.Get(SSLCertPathConfig); certPath != "" {
```

Comment:
It should not be possible to specify both cert and cert path, can you add a condition to check it and return an actionable error message? And a test.

Reply:
Sure, i will add condition for this